subworkflows/local/prepare_input_stitch/prepare_posfile_tsv.nf

```diff
@@ -0,0 +1,26 @@
+include { BCFTOOLS_QUERY                    } from '../../../modules/nf-core/bcftools/query/main'
+include { GAWK                              } from '../../../modules/nf-core/gawk'
+
+
+workflow PREPARE_POSFILE_TSV {
+
+    take:
+    ch_panel_sites
+    ch_fasta
+
+    main:
+
+    ch_versions      = Channel.empty()
+
+    // Convert position file to tab-separated file
+    BCFTOOLS_QUERY(ch_panel_sites, [], [], [])
+    ch_posfile = BCFTOOLS_QUERY.out.output
+
+    // Remove multiallelic positions from tsv
+    GAWK(ch_posfile, [])
```

Comment:
I don't know if it's better or not to give the script in the config file directly or in a file with collectFile()...
We might need to normalize this usage with GAWK.

Reply:
Sarek also gives the script in the config file directly (example here: https://github.com/search?q=repo%3Anf-core%2Fsarek%20BUILD_INTERVALS&type=code). I've found this method more straightforward.

Reply:
Okay, so I'll update that for the other tools.
I've add that as an issue.